### PR TITLE
Improves the Dark Gygax Slightly

### PR DESCRIPTION
**Pull request by SchrodingersWolf**
## What Does This PR Do

Dark Gygax now is preequipped with a Syndicate Scattershot that fires a shotgun spread of C20r bullets (20 BRUTE, 45 STAMINA, compared to 25 BRUTE previously). Crew Scattershot remains unchanged. (Though this might change in the future)

Dark Gygax also has it's TC cost reduced - 400 TC to 350 TC now. Hopefully this will allow the mech pilot to at least get some additional backup gear.

Also removes the flavour text in the Dark Gygax Uplink description about it not having thrusters, as it was changed in #13332 - That was in 2020 and we *only* discovered it now? God, really nobody uses mechs on nukies.

## Why It's Good For The Game

Mechs on nukies are always underwhelming, and nothing is more disappointing than Dark Gygax - It has very little health for a mech (300 intergrity), a weak shotgun with a large spread that does 25 damage per pellet (For reference WTs do 20 damage), and on top of that you are really vulnerable to EMPs/ions while not being able to parry and have to contend with malfunctions.

Even in #16975, the author mentioned that the weapon is problematic in crew hands, not in nukies: https://github.com/ParadiseSS13/Paradise/pull/16975#issuecomment-950488643

Compared to Elite Hardsuits, which are EMP proof and have extremely good armour as well, the value proposition just isn't there. Hopefully by reducing the TC a bit and giving Dark Gygax back its stamina damage it could do better.

If it doesn't then I might just give them the AC-2 as well.

False advertising making the Dark Gygax sound even worse is just another nail in the coffin and we should probably remove it.

## Images of changes

Lazy, sorry.

## Testing

It compiles fine.
Booted up the game, spawned in a nukie uplink, verified it is 350 TC.
Bought the Dark Gygax, spawned a few skrells. Shot at them point blank and then aghosted into them, verified that I am indeed stamcritted.

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>
image
image

## Changelog
:cl:
add: A Syndicate scattershot module that deals stamina and brute damage
tweak: Dark Gygax now costs 350 TC and has the new Syndicate Scattershot module
fix: Dark Gygax's Uplink Description is now changed to accurately reflect its capabilities
/:cl:

### CODE REVIEW

**Comment by SchrodingersWolf** [comment]
New build changed the description of the Dark Gygax's Uplink description - See updated PR description and changelog for changes.

I had wanted to add this earlier, but I forgot to do so before I pushed my build out

**Review by Burzah — APPROVED** [review]

**Review by DGamerL — APPROVED** [review]

File: code/game/mecha/equipment/weapons/weapons.dm [hunk 1]
```diff
@@ -317,6 +317,19 @@
 	variance = 25
 	harmful = TRUE
 
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/scattershot/syndie //Version used for Dark Gygax
+	equip_cooldown = 2 SECONDS
```
Comment (DGamerL) [inline code/game/mecha/equipment/weapons/weapons.dm:321]: This should be at the bottom of all variables.
Name and description always go first.

File: code/game/mecha/equipment/weapons/weapons.dm [hunk 2]
```diff
@@ -317,6 +317,19 @@
...
+	name = "\improper LBX AC 20-r \"Scattershot .45\""
+	icon_state = "mecha_scatter"
+	origin_tech = "combat=4;syndicate=2" //Crew is not going to get it normally anyways
+	projectile = /obj/item/projectile/bullet/midbullet
+	fire_sound = 'sound/weapons/gunshots/gunshot_shotgun.ogg'
+	projectiles = 40
+	projectile_energy_cost = 25
+	projectiles_per_shot = 4
+	variance = 25
+	harmful = TRUE
```
Comment (DGamerL) [inline code/game/mecha/equipment/weapons/weapons.dm:331]: These are all copied from the normal scattershot and are unnecessary

**Review by DGamerL — CHANGES_REQUESTED** [review]
Please remove all unnecessary variables on the syndie version!

File: code/game/mecha/equipment/weapons/weapons.dm (hunk 1 above)
Comment (SchrodingersWolf) [reply code/game/mecha/equipment/weapons/weapons.dm:321]: Edited - Basetype has it too so I'm removing it

File: code/game/mecha/equipment/weapons/weapons.dm (hunk 2 above)
Comment (SchrodingersWolf) [reply code/game/mecha/equipment/weapons/weapons.dm:331]: Removed unnecessary variables - Some are actually different so those stay

**Review by DGamerL — APPROVED** [review]

**Pull request merged**